dm/syncer/syncer.go

```diff
@@ -431,13 +432,6 @@ func (s *Syncer) Init(ctx context.Context) (err error) {
 			}
 		}
 	}
-	s.checkpointFlushWorker = &checkpointFlushWorker{
```

Comment:
all wokrer's lifecycle  (dml/ddl/checkpointWorker) was moved into `s.Run` and use `s.runCtx` `s.runTctx` `s.runwg` to control

s.Run will make sure all background jobs which created by itself will be closed before it return

Reply:
dml/ddl worker is not a member of syncer, I think the member initialization of syncer should be put in `Syncer.Init`